packages/http-client-java/generator/http-client-generator-test/src/main/java/azure/resourcemanager/operationtemplates/fluent/LroesClient.java

```diff
@@ -9,47 +9,12 @@
 import azure.resourcemanager.operationtemplates.models.ExportRequest;
 import com.azure.core.annotation.ReturnType;
 import com.azure.core.annotation.ServiceMethod;
-import com.azure.core.management.polling.PollResult;
 import com.azure.core.util.Context;
-import com.azure.core.util.polling.SyncPoller;
 
 /**
  * An instance of this class provides access to all the operations defined in LroesClient.
  */
 public interface LroesClient {
-    /**
-     * Create a Order.
-     * 
-     * @param resourceGroupName The name of the resource group. The name is case insensitive.
-     * @param orderName The name of the Order.
-     * @param resource Resource create parameters.
-     * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @throws com.azure.core.management.exception.ManagementException thrown if the request is rejected by server.
-     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return the {@link SyncPoller} for polling of concrete tracked resource types can be created by aliasing this
-     * type using a specific property type.
-     */
-    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    SyncPoller<PollResult<OrderInner>, OrderInner> beginCreateOrReplace(String resourceGroupName, String orderName,
-        OrderInner resource);
```

Comment:
Why these API are gone? I assume this does not enable sync-stack, so the interface should not change?

I think we didn't expose the `begin` API in convenience code, but user can still call here to do `begin` of the LRO.

Reply:
Yeah, this shouldn't be gone.. It should a regression during a recent fix in method visibility in `FluentMethodMapper`. Let me check..